insolar/insolar
# Ins 1113 trace using jaeger

How to test:

1. Start Jaeger:
```
docker run --rm --name jaeger -p 6831:6831/udp -p 16686:16686 jaegertracing/all-in-one:1.8
```
2. Run launchnet.sh:
```
./scripts/insolard/launchnet.sh -g
```
3. Run a benchmark:
```
while true; do ./bin/benchmark -c=1 -r=50 -k=scripts/insolard/configs/root_member_keys.json; done
```
4. Open http://localhost:16686 to see traces
